### PR TITLE
TE-9607 Covered code with tests

### PR DESCRIPTION
**Pull request by geega**

### CODE REVIEW

File: src/SprykerSdk/Integrator/Business/Manifest/ManifestReader.php [hunk 1]
```diff
@@ -73,12 +76,17 @@ public function readManifests(array $moduleTransfers): array
...
     {
         $recipesArchive = $this->config->getRecipesDirectory() . 'archive.zip';
+
         if (!is_dir($this->config->getRecipesDirectory())) {
             mkdir($this->config->getRecipesDirectory(), 0700, true);
         }
-        file_put_contents($recipesArchive, fopen($this->config->getRecipesRepository(), 'r'));
 
-        $zip = new \ZipArchive;
+        var_dump($recipesArchive);
```
Comment (gechetspr) [inline src/SprykerSdk/Integrator/Business/Manifest/ManifestReader.php]: leftovers

File: src/SprykerSdk/Integrator/Business/ManifestStrategy/CopyModuleFileManifestStrategy.php [hunk 2]
```diff
@@ -63,9 +63,9 @@ protected function getSourcePath(string $source, string $moduleName): string
         [$organisation, $moduleName] = explode('.', $moduleName);
 
         return $this->config->getCoreRootDirectory()
-            . $this->camelCaseToDash($organisation)
+            . strtolower($this->camelCaseToDash($organisation))
```
Comment (gechetspr) [inline src/SprykerSdk/Integrator/Business/ManifestStrategy/CopyModuleFileManifestStrategy.php:66]: it's always good idea to use mb_* functions

File: src/SprykerSdk/Shared/Common/AbstractConfig.php [hunk 3]
```diff
@@ -10,6 +15,8 @@ abstract class AbstractConfig
     protected static $instance;
 
     /**
+     * @api
```
Comment (gechetspr) [inline src/SprykerSdk/Shared/Common/AbstractConfig.php:18]: no need for api tags for a library

File: src/SprykerSdk/Shared/ModuleFinder/ModuleFinderFactoryAwareTrait.php [hunk 4]
```diff
@@ -1,6 +1,11 @@
 <?php
 
-declare(strict_types=1);
+declare(strict_types = 1);
```
Comment (gechetspr) [inline src/SprykerSdk/Shared/ModuleFinder/ModuleFinderFactoryAwareTrait.php:3]: it's without spaces in official docs

File: tests/bootstrap.php [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<?php
```
Comment (gechetspr) [inline tests/bootstrap.php:1]: why do we need this file?

**Review by gechetspr — CHANGES_REQUESTED** [review]
Looks good, but there are a few things to fix.

Also a lot of ToDos in the code

File: tests/bootstrap.php (hunk 5 above)
Comment (geega) [reply tests/bootstrap.php:1]: To define constants for run PHPUnit only.

**Pull request merged**